README.md

```diff
@@ -508,17 +507,19 @@ Read more at [Proguard](https://www.guardsquare.com/en/proguard/manual/examples)
 
 #### SharedPreferences
```

Comment:
True. In my opinion, this section needs a rewrite because it doesn't acknowledge the middle ground between SharedPreferences and ContentProviders. ContentProviders have their place, but I'm not sure it makes sense to heavily recommend them when other, simpler options often suffice.